templates/deployment.yaml

```diff
@@ -176,23 +183,28 @@ spec:
           {{- else }}
           secretName: {{ template "integration-manager-worker.fullname" . }}-gcp
           {{- end }}
-      - name: worker-config
+      - name: worker-shared-volume
         configMap:
-          name: {{ template "integration-manager-worker.fullname" . }}
+          name: {{ template "integration-manager-worker.fullname" . }}-shared
```

Comment:
Why did this configmap reference need to be changed? The configmap is provided directly from this chart and doesn't include the '-shared' suffix